source/fundamentals/transactions.txt

```diff
@@ -208,6 +208,15 @@ the Convenient Transaction API:
 To see a fully runnable example that uses this API, see the
 :ref:`node-usage-convenient-txn` usage example.
 
+.. sharedinclude:: dbx/transactions-parallelism.rst
+
+   .. replacement:: driver-specific-content
+
+      If you're using {+mdb-server+} v8.0 or later, you can perform
+      write operations on multiple namespaces within a single transaction by using
+      the ``bulkWrite()`` method. For more information,
```

Comment:
I would change it to say client.bulkWrite since there's already the collection.bulkWrite that does not work across namespaces (given its nesting under the collection object) so we should disambiguate

Reply:
The link in the for more information links to the collection API:

https://deploy-preview-952--docs-node.netlify.app/usage-examples/bulkWrite/#std-label-node-usage-bulk

> The bulkWrite() method performs batch write operations against a single collection.

Reply:
Ah ok updating. I think the correct page to link to is one that is [in progress](https://github.com/mongodb/docs-node/pull/948) actually so will add the link for that once published.

Reply:
Looking over it again, I'll just leave out this mention as we did with the PyMongo docs that also don't have this page yet and if needed can be added after the docs are available.